pystemd/dbusc.pxd

```diff
@@ -217,3 +206,6 @@ cdef union basic_data:
 
 cdef extern from 'stdbool.h':
   pass
+
+IF LIBSYSTEMD_VERSION >= 237:
+  include "dbusc_v237.pxi"
```

Comment:
it consern me that the conservative option (when LIBSYSTEMD_VERSION is not defined) is to not include the file, think the right options here should be:

If LIBSYSTEMD_VERSION is not defined or if its defined and its >= 237 include the file.

the reason: many folks use other build systems other than setup.py to get this into their libraries (e.g. some use buck and bzl). and we dont wanna break them.